examples/nl-core-Procedure-event-03.xml

```diff
@@ -63,4 +63,11 @@
          <display value="structuur van knieregio"/>
       </coding>
    </bodySite>
+   <focalDevice>
+      <manipulated>
+         <reference value="DeviceUseStatement/nl-core-Procedure-01-DeviceUseStatement-01"/>
+         <type value="DeviceUseStatement"/>
+         <display value="medisch_hulpmiddel"/>
+      </manipulated>
+   </focalDevice>
```

Comment:
This reference is not possible because the resource type must be Device rather than DeviceUseStatement

See specs: https://hl7.org/fhir/r4/procedure.html